src/pages/FeelGoodPage.js

```diff
@@ -13,16 +17,29 @@ export default function FeelGoodPage() {
           width="60"
         />
       </GoBackButton>
-      <Header>Scared?</Header>
-      <Text>
-        Don't worry!
-        <br />
-        The monsters under your bed are not that bad...
-        <br />
-        You should be more worried about the monsters in your closet!
-      </Text>
+      <Header>Scared? Not anymore!</Header>
+      <CatContainer>
+        <img
+          src={catUrl?.url}
+          alt="Press the button below to see cute cats!"
+          width="300px"
+        />
+        <PinkButton onClick={fetchCatImages}>Press for a cute cat!</PinkButton>
+      </CatContainer>
     </Flex>
   );
+
+  async function fetchCatImages() {
+    try {
+      const response = await fetch(
+        'https://api.thecatapi.com/v1/images/search'
+      );
+      const data = await response.json();
+      setCatUrl(data[0]);
+    } catch (error) {
+      console.log(error);
```

Comment:
I would remove the console.log and would add an error state to it, here is an example:   const fetchData = useCallback(async () => {
    try {
      const response = await fetch(url);
      const receivedData = await response.json();
      setLoading(false);
      setData(receivedData);
    } catch (error) {
      setError(error.message);
      setLoading(false);
    }
  }, [url]);